### ISSUE TITLE
Feature request: skipif(Command('hostname -s') == 'druid'))

### ISSUE THREAD

**Issue by emayssat-ms** [philpep/testinfra#129]
Is it possible to skip, pass, xfail a test based on a REMOTE command/parameter?
If so, how?

**Comment by philpep** [comment]
```python
import pytest

def test_foo(Command):
    if Command.check_output('hostname -s') == 'druid':
        pytest.skip()

```

?

**Comment by philpep** [comment]
Closed in flavor of #48
